### PR TITLE
Added tabTest for close tab and reopen closed tab

### PR DESCRIPTION
**Pull request by rob-blasetti**

### CODE REVIEW

File: test/components/tabTest.js [hunk 1]
```diff
@@ -63,6 +63,32 @@ describe('tabs', function () {
...
+      yield setup(this.app.client)
+    })
+    it('should close the tab', function *() {
+      yield this.app.client
+        .ipcSend(messages.SHORTCUT_NEW_FRAME, 'https://www.brave.com')
+        .waitForExist('.tab[data-frame-key="2"]')
+        .ipcSend(messages.SHORTCUT_CLOSE_FRAME)
+
+      yield this.app.client
+        .isExisting('.tab[data-frame-key="2"]').should.eventually.be.false
```
Comment (bbondy) [inline test/components/tabTest.js]: I think we've found that should.eventually.be is very intermittent error prone. Could you change those to waitUntil equivalents?

Comment (bbondy) [inline test/components/tabTest.js]: Otherwise looks good, thanks.

**Comment by rob-blasetti** [comment]
@bbondy How are these updated tests?

**Comment by bbondy** [comment]
sorry for the delay on this was swamped with the release. I'll take a look early this week.

**Comment by bbondy** [comment]
Looks good, thanks!

**Pull request merged**